packages/editor/src/lib/editor/tools/StateNode.ts

```diff
@@ -47,6 +47,7 @@ export abstract class StateNode implements Partial<TLEventHandlers> {
 		public editor: Editor,
 		parent?: StateNode
 	) {
+		// eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
This this one should be fine as it is done in the constructor so it should be bound to the correct thing.